src/lib.rs

```diff
@@ -612,7 +612,7 @@ mod cache_tests {
             }
             Ok(())
         }
-        let mut paths = vec!();
+        let mut paths = vec![];
```

Comment:
Why this change?

Reply:
For style, generally vec! uses square brackets because it is used the same as the array initializer (`[a, b, c]`)